sql/data_model.sql

```diff
@@ -0,0 +1,64 @@
+BEGIN;
+
+-- add pgcrypto extension so that we can use the gen_random_uuid function.
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+DROP TABLE IF EXISTS formdata CASCADE;
+CREATE TABLE formdata(
+  id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+  domain text NOT NULL,
+  received_on timestamp without time zone,
+  time_start timestamp without time zone,
+  time_end timestamp without time zone,
+  duration bigint,
+  device_id text,
+  user_id uuid NOT NULL,
+  username text,
+  app_id uuid,
+  xmlns text NOT NULL,
+  form_json text NOT NULL
```

Comment:
very minor, but postgres has a json field that we could consider using for this. i think it'll help with data integrity as it ensures that the data is valid json